.github/pull_request_template.md

```diff
@@ -55,5 +55,6 @@ Select one or more:
 - [ ] I have tested my changes using the available tooling
 - [ ] I have commented my code
 - [ ] I have performed a self-review of my own code; both comments & source code
+    - [ ] I have **thoroughly** reviewed any on-chain business logic which may result in non-deterministic serialization of repeated fields
```

Comment:
Thoughts on adding a small paragraph on it here in [1] and linking to it here?

- Call out the tool you built
- Explain that on-chain implies `./x/...` and any deps it imports
- Etc...

```suggestion
- [ ] I have performed a self-review of my own code; both comments & source code
- [ ] I have **thoroughly** reviewed any on-chain business logic which may result in non-deterministic serialization of repeated fields
```

[1] https://dev.poktroll.com/develop/contributing/code_review_guidelines